lib/asset/transferasset.js

```diff
@@ -150,6 +152,26 @@ class TransferAsset {
         return this[PRIVATE].multipartTarget;
     }
 
+    /**
+     * Create a file event information for events sent by the module.
+     *
+     * @returns {*} File event data.
+     */
+    get eventData() {
+        const data = {
+            fileName: decodeURI(urlPathBasename(this.target.url.pathname)),
+            fileSize: this.metadata.contentLength,
+            targetFolder: decodeURI(urlPathDirname(this.target.url.pathname)),
+            targetFile: decodeURI(this.target.url.pathname)
+        };
+
+        if (this.metadata.contentType) {
+            data.mimeType = this.metadata.contentType;
```

Comment:
Do you need miemetype always set? If so you could set `application/octet-stream` as default.